pyproject.toml

```diff
@@ -3,6 +3,31 @@ name = "searvey"
 version = "0.1.0"
 description = ""
 authors = ["Panos Mavrogiorgos <pmav99@gmail.com>"]
+license = "EUPL-1.0"
+readme = "README.md"
+repository = "https://github.com/oceanmodeling/searvey.git"
+classifiers = [
+    "License :: OSI Approved :: European Union Public Licence 1.2 (EUPL 1.2)",
+    "Operating System :: OS Independent",
+    "Development Status :: 4 - Beta",
+    "Environment :: Other Environment",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Atmospheric Science",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+]
+
+[build-system]
+requires = [
+    "poetry-core>=1.0.0",
+    "poetry-dynamic-versioning",
+]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry-dynamic-versioning]
+enable = true
```

Comment:
`poetry-dynamic-versioning` builds a version number from the latest Git tag (plus `.post*` commit suffix) using `dunamai` as a backend